src/Neo.Json/Neo.Json.csproj

```diff
@@ -1,9 +1,15 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <ImplicitUsings>enable</ImplicitUsings>
-    <Nullable>enable</Nullable>
-    <PackageTags>NEO;JSON</PackageTags>
-  </PropertyGroup>
-
-</Project>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.1;net7.0</TargetFrameworks>
+    <LangVersion>10.0</LangVersion>

```

Comment:
.NET Standard 	2.1 	requires Lang 8.0, right?
And .NET 	7.x 	C# 11? 

is Lang 10 fine?

Reply:
We use 11 in BLS and it works, we should use in all the same